src/CameraScreen/CameraKitCameraScreenBase.js

```diff
@@ -21,8 +21,8 @@ const FLASH_MODE_AUTO = 'auto';
 const FLASH_MODE_ON = 'on';
 const FLASH_MODE_OFF = 'off';
 const OVERLAY_DEFAULT_COLOR = '#ffffff77';
-const OFFSET_FRAME = 30;
-const FRAME_HEIGHT = 200;
+const FRAME_LEFT = 0;
+const FRAME_TOP = 0;
```

Comment:
What does it mean? margin/padding top? top frame?

Reply:
this constant similar to value that we sets frameLeft, frameTop